### DISCUSSION TITLE
How to hide HA Toolbar for a wallpanel

### DISCUSSION THREAD

**Discussion by Morsmorse** [matt8707/ha-fusion#513] [Q&A] [answered]
Hello, 

I want to use HA Fusion on my tablet wall-panel. I don't wand nor need to access other Homeassistant function via fully kiosk (I can switch to HA-App if I really need to). 

How do i hide the HA toolbar? I tried installing kiosk mode and wallpanel via HACS but they don't work in HA Fusion. Am I missing something here? I didn't find anything related in the docs.

Does someone have a solution for that?

**Comment by matt8707** [comment]
https://github.com/matt8707/ha-fusion/issues/453

**Reply by Morsmorse** [reply]
Hello Matt, thanks for getting back to me and linking that issue. I don't want to hide the menu, views or header. I want to hide the default homeassistant toolbar on the left side. This one: 
![image](https://github.com/matt8707/ha-fusion/assets/31167754/2cda9d32-f543-41bf-90d3-20442c4c75cf)

I read the linked issue multiple times, I can't finde a solution for this there.

**Reply by Morsmorse** [reply, marked as answer]
Hello Matt! I just realized, I have to expose the port in the add-on settings and then use the homeassistant.local:5050 (or whatever you exposed) to display the dashboard without the homeassistant toolbar. I was using the (internal) url to the addon before. 

I'm happy to get fully into it now! Thanks for providing HA Fusion!
